.meta

```diff
@@ -0,0 +1,10 @@
+{
+  "projects": {
+    "cache": "git@github.com:rsksmart/rif-marketplace-cache.git",
+    "dapp": "git@github.com:rsksmart/rif-marketplace-ui.git",
+    "pinner": "git@github.com:rsksmart/rif-storage-pinner.git",
+    "upload-service": "git@github.com:rsksmart/rif-marketplace-upload-service.git",
+    "rns-manager": "git@github.com:rnsdomains/rns-manager-react.git",
+    "notifier": "git@github.com:rsksmart/rif-notifier.git"
+  }
```

Comment:
comms? https://github.com/rsksmart/rif-communications-pubsub - has been (though I think it isn't at the moment (, which brings me to a question: does the storage work as expected right now?)) used for communication between the upload service, the pinner and the cache. it could also be used to communicate stuff between notifier and the cache, if we were to follow the defined architecture (the reason this has been done so for the storage)

Reply:
it's not used right now, but if we will support the project in the future, we can enable it. I also mentioned in the readme  that it's not used.